pkgs/native_assets_cli/lib/src/api/link_config.dart

```diff
@@ -31,6 +32,11 @@ abstract class LinkConfig implements HookConfig {
   /// `build.dart` script destined for this packages `link.dart`.
   Iterable<Asset> get assets;
 
+  /// The file containing the recorded usages, serialized by
+  /// `package:record_use`.
+  @experimental
+  Uri? get recordedUsages;
```

Comment:
Let's rename it to something that makes it clear from name that it's a file (e.g. `recordedUsagesFile`)